hardware_interface/include/hardware_interface/force_torque_sensor_interface.h

```diff
@@ -40,7 +40,7 @@ namespace hardware_interface
 class ForceTorqueSensorHandle
 {
 public:
-  ForceTorqueSensorHandle() {}
+  ForceTorqueSensorHandle() = default;
 
```

Comment:
I was referring to this one.

Reply:
All the `XYZHandle`s have a default constructor and parameterized constructor, so I just left the default ctors to avoid breaking API/ABI.

Reply:
I see now. Thanks!